editor/property_selector.cpp

```diff
@@ -343,15 +343,29 @@ void PropertySelector::_update_search() {
 	get_ok_button()->set_disabled(root->get_first_child() == nullptr);
 }
 
+Dictionary selected_properties;
```

Comment:
This should not be added outside the function, it should be in the header

Reply:
Fixed in latest commit